UniConnectApp/app/src/main/java/com/example/uniconnect/AppModule.kt

```diff
@@ -1,6 +1,6 @@
 package com.example.uniconnect
 
-import com.example.uniconnect.service.IUniversityService
+import com.example.uniconnect.dao.IUniversityService
```

Comment:
A service interface belongs in a service package, not a dao package.  I'd leave this as it were.